src/neuroconv/datainterfaces/ecephys/spikeinterface/sipickledatainterfaces.py

```diff
@@ -17,6 +17,7 @@ def __init__(self, file_path: FilePathType, verbose: bool = True):
         self.subset_channels = None
         self.source_data = dict(file_path=file_path)
         self.verbose = verbose
+        self.es_key = None
```

Comment:
Temporary fix here since this interface does not call `super`.

Also Alessio said they no longer use this and would be OK to remove, so I'll do that in a follow-up